src/main/java/it/gov/pagopa/payhub/auth/controller/AuthzControllerImpl.java

```diff
@@ -52,6 +53,11 @@ public ResponseEntity<OperatorsPage> getOrganizationOperators(String organizatio
                 .build());
     }
 
+    @Override
+    public ResponseEntity<UserInfo> getUserInfoFromMappedExternaUserId(String mappedExternalUserId) {
+        return ResponseEntity.ok(authzService.getUserInfoFromMappedExternalUserId(mappedExternalUserId));
```

Comment:
This API should be allowed to users having ROLE_ADMIN role to at least one organization

Reply:
allow also the loggedUser to access to its data